tests/builder/Data/ByteString/Builder/Tests.hs

```diff
@@ -637,333 +641,353 @@ testsASCII =
+          , ( (1/0)  , "Infinity" )
+          , ( (-1/0) , "-Infinity" )
+          ]
+    , testMatches "d2sSubnormal" doubleDec show
+          [ ( 2.2250738585072014e-308 , "2.2250738585072014e-308" )
+          ]
+    , testMatches "d2sMinAndMax" doubleDec show
+          [ ( (coerceWord64ToDouble 0x7fefffffffffffff) , "1.7976931348623157e308" )
+          , ( (coerceWord64ToDouble 0x0000000000000001) , "5.0e-324" )
+          ]
+    , testMatches "d2sTrailingZeros" doubleDec show
+          [ ( 2.98023223876953125e-8 , "2.9802322387695313e-8" )
+          ]
+    , testMatches "d2sRegression" doubleDec show
+          [ ( (-2.109808898695963e16) , "-2.1098088986959632e16" )
+          , ( 4.940656e-318           , "4.940656e-318" )
+          , ( 1.18575755e-316         , "1.18575755e-316" )
+          , ( 2.989102097996e-312     , "2.989102097996e-312" )
+          , ( 9.0608011534336e15      , "9.0608011534336e15" )
+          , ( 4.708356024711512e18    , "4.708356024711512e18" )
+          , ( 9.409340012568248e18    , "9.409340012568248e18" )
+          , ( 1.2345678               , "1.2345678" )
+          , ( 1.9430376160308388e16   , "1.9430376160308388e16" )
+          , ( (-6.9741824662760956e19), "-6.9741824662760956e19" )
+          , ( 4.3816050601147837e18   , "4.3816050601147837e18" )
+          ]
+    , testExpected "d2sScientific" (formatDouble scientific)
+          [ ( 0.0         , "0.0e0"         )
+          , ( 1.2345678   , "1.2345678e0"   )
+          , ( 4.294967294 , "4.294967294e0" )
+          , ( 4.294967295 , "4.294967295e0" )
+          ]
+    , testGroup "d2sStandard"
+      [ testCase "specific" do
+          singleMatches (formatDouble (standard 2)) (flip (showFFloat (Just 2)) []) ( 12.3    , "12.30"    )
+          singleMatches (formatDouble (standard 2)) (flip (showFFloat (Just 2)) []) ( 12.345  , "12.34"    )
+          singleMatches (formatDouble (standard 2)) (flip (showFFloat (Just 2)) []) ( 12.3451 , "12.35"    )
+          singleMatches (formatDouble (standard 2)) (flip (showFFloat (Just 2)) []) ( 0.0050  , "0.00"     )
+          singleMatches (formatDouble (standard 2)) (flip (showFFloat (Just 2)) []) ( 999.999 , "1000.00"  )
+          singleMatches (formatDouble (standard 2)) (flip (showFFloat (Just 2)) []) ( 999.199 , "999.20"   )
+          singleMatches (formatDouble (standard 2)) (flip (showFFloat (Just 2)) []) ( 0.999   , "1.00"     )
+          singleMatches (formatDouble (standard 2)) (flip (showFFloat (Just 2)) []) ( 0.0051  , "0.01"     )
+          singleMatches (formatDouble (standard 3)) (flip (showFFloat (Just 3)) []) ( 0.0056  , "0.006"    )
+          singleMatches (formatDouble (standard 3)) (flip (showFFloat (Just 3)) []) ( 0.0096  , "0.010"    )
+          singleMatches (formatDouble (standard 5)) (flip (showFFloat (Just 5)) []) ( 12.345  , "12.34500" )
+          singleMatches (formatDouble (standard 3)) (flip (showFFloat (Just 3)) []) ( 0.0     , "0.000"    )
```

Comment:
Since this one is currently broken, split it out and `expectFailure` it so that CI doesn't red while the fix is un-merged.